matlab/test/arrow/tabular/tRecordBatch.m

```diff
@@ -494,9 +494,23 @@ function TestIsEqualFalse(testCase)
             % Call isequal with more than two arguments
             testCase.verifyFalse(isequal(rb1, rb2, rb3, rb4));
         end
-
-
+       function testArrowRecordBatchNoArgs()
+    % Call arrow.recordBatch with no arguments
+    actual = arrow.recordBatch();
+    
+    % Create an empty expected RecordBatch
+    schema = arrow.schema();
+    expected = arrow.tabular.RecordBatch(schema, {});
```

Comment:
The `arrow.tabular.RecordBatch` constructor does not support any syntax that accepts an `arrow.tabular.Schema` and an empty `cell` array (i.e. `{}`).